jobs/config.json

```diff
@@ -5383,9 +6568,13 @@
     "args": [
       "--env-file=jobs/platform/gke.env",
       "--provider=gke",
+      "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
+      "--provider=gke",
       "--gcp-zone=us-west1-b",
       "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
       "--env-file=jobs/ci-kubernetes-e2e-gke-gpu.env",
+      "--gcp-project=k8s-jkns-e2e-gke-gpus",
+      "--gcp-zone=us-west1-b",
```

Comment:
This is a newly added job (added couple of hours ago), I added it using the new format already, so these flags are duplicate now. And one of them (`--gcp-project`) is incorrect.